app/backend/lib/excel_import/community_progress_report.ts

```diff
@@ -165,7 +165,7 @@ const ValidateData = async (data, applicationRowId, req) => {
 
   data.communityData.forEach((comData) => {
     if (
-      comData.communityName === undefined ||
```

Comment:
@AntBush This PR looks good to me, though I thought we were removing these validations all together other than the CCBC ID?

Reply:
I assumed we still wanted type validations (and the validation for the summary at the top). This right now is checking that if it's not empty then the type is correct. Let me check in with Mars though

Reply:
Checked in Mars, it's all good 👍